app/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/demographics/ConvertAndSendLabOrderUsecaseTest.groovy

```diff
@@ -19,6 +19,16 @@ class ConvertAndSendLabOrderUsecaseTest extends Specification {
                     String getUnderlyingOrder() {
                         return "This is a mock inner order"
                     }
+
+                    @Override
+                    String getFhirResourceId() {
+                        return null
+                    }
+
+                    @Override
+                    String getPatientId() {
+                        return null
+                    }
                 }
```

Comment:
Maybe this can be converted to use the `OrdersMock` (or `LabOrderMock` if you take that suggestion) class that was made in this PR.